slsim/lens_pop.py

```diff
@@ -372,7 +382,7 @@ def deflector_number(self):
 
         :return: number of potential deflectors
         """
-        return self._lens_galaxies.deflector_number()
+        return round(self.factor * self._lens_galaxies.deflector_number())
```

Comment:
is there also a deflector_number_selected() argument? Perhaps we missed that here.

Reply:
In deflector case, self.deflector_number() is the selected number.